doc/nrf/ug_bt_fast_pair.rst

```diff
@@ -85,7 +85,10 @@ The Fast Pair service implementation provides API to generate the advertising da
   This function is to be used to set pairing mode before the advertising is started.
 
 Since you control the advertising, make sure to use advertising parameters consistent with the specification.
-The Bluetooth privacy is selected by the Fast Pair service, but you must make sure that the private address rotation is synchronized with the advertising payload update during the not discoverable advertising.
+The Bluetooth privacy is selected by the Fast Pair service, but you must make sure that:
```

Comment:
```suggestion
The Bluetooth privacy is selected by the Fast Pair service, but you must make sure that the following points are addressed:
```

Reply:
Added: `... the following requirements are met` (I think these are not changes). Please rereview